specification/resources/resource-manager/readme.md

```diff
@@ -57,19 +57,10 @@ tag: package-managedapplications-2018-06
 ```
 
 ``` yaml $(package-deploymentscripts)
-tag: package-2019-11
+tag: package-deploymentscripts-2019-10-preview
 ```
 
-
-### Tag: package-2019-11
-
-These settings apply only when `--tag=package-2019-11` is specified on the command line.
-
-```yaml $(tag) == 'package-2019-11'
-input-file:
-  - Microsoft.Resources/stable/2019-11-01/subscriptions.json
-```
-### Tag: package-resources-2019-10-preview
+### Tag: package-deploymentscripts-2019-10-preview
```

Comment:
Fix a [mismatch](https://github.com/Azure/azure-rest-api-specs/pull/7486/files#diff-d474de52c790d7028fb81d4abf0b9618R63) made in https://github.com/Azure/azure-rest-api-specs/pull/7486. Let `package-deploymentscripts-2019-10-preview` in the title match that in the content.

Reply:
This was intentional however making naming consistent in tags makes sense.